tensorflow/core/grappler/optimizers/auto_mixed_precision_lists.h

```diff
@@ -132,6 +132,9 @@ class AutoMixedPrecisionListsCuda : public AutoMixedPrecisionLists {
         "LSTMBlockCell",
         "LSTMBlockCellGrad",
         "MatMul",
+        // `TRTEngineOp` needs to be added here to protect the OP in case the
+        // user execute another pass of mixed_precision after TF-TRT conversion.
```

Comment:
s/execute/executes/

how can the user executes precision after TF-TRT conversion? why do we need to consider such a scenario?

Reply:
Multiple ways.

- Try to re-convert an already converted SavedModel
- Importing a converted SavedModel (`converter.save()`) and executing some grappler optimizations (namely `auto_mixed_precision`)